Cython/Includes/cpython/unicode.pxd

```diff
@@ -111,6 +130,18 @@ cdef extern from *:
     # UTF-8 encoded bytes.  The size is determined with strlen().
     unicode PyUnicode_FromString(const char *u)
 
+    unicode PyUnicode_New(Py_ssize_t size, Py_UCS4 maxchar)
+    unicode PyUnicode_FromKindAndData(int kind, const void *buffer, Py_ssize_t size)
+    unicode PyUnicode_FromFormat(const char *format, ...)
+    Py_ssize_t PyUnicode_GetLength(object unicode) except -1
+    Py_ssize_t PyUnicode_CopyCharacters(object to, Py_ssize_t to_start, object from, Py_ssize_t from_start, Py_ssize_t how_many) except -1
```

Comment:
`from` is a keyword so can't be an argument name.

Reply:
We try to use the documented argument names to help with keyword argument calls, but if we can't, we either chose similar names (which I tend to prefer) or an underscore as suffix. We use `pfrom` and `pto` for the pointers in `memcpy()` and friends, for example, although that wouldn't fit here.